convert-twiki

```diff
@@ -1,11 +1,25 @@
 #!/bin/bash
+set -e
+set -o pipefail
 
 TWIKI=$1
 MARKDOWN=$2
 
 TITLE=$(basename "$TWIKI")
 ARCHIVE=archive/$TITLE
 
+tmo_msg () (
+  set +e
```

Comment:
Does this only apply to the function scope? aka, when the function completes, does it go back to `set -e`?

Reply:
Sorry I originally made the change in an additional commit, 58e459b, where it was probably easier to see, then rebased this branch.

Yes, and yes.  I replaced the braces `{}` with parens `()` in the function to make it a subshell, so any `set` options are local to the function (which runs in a separate process).  Even this is actually redundant in this script, since it is running as part of a pipeline (because every command in a pipeline is automatically run in a subshell/subprocess), but for the sake of readability, and in case we decide to do away with the pipelines, the function is now run in a `()` subshell.